catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/BackupCommand.java

```diff
@@ -148,7 +147,7 @@ private void performSolrCloudBackup() throws IOException {
     }
   }
 
-  private void printResponse(@Nullable HttpResponse response) {
+  private void printResponse(HttpResponse response) {
```

Comment:
❓ Why remove the annotation? Is the response no longer nullable?

Reply:
Because it was forcing a dependency on guava for an unnecessary annotation.